web/server/vue-cli/package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "@mdi/font": "^5.3.45",
-    "codechecker-api": "6.37.0",
+    "codechecker-api": "6.37.0-dev6",
```

Comment:
You have to commit the `package-lock.json` file too. Just run `npm install` in the `vue-cli` directory and it will update this file.